igvm/commands.py

```diff
@@ -202,7 +202,7 @@ def vm_migrate(vm_hostname, hypervisor_hostname=None, newip=None,
         vm = es.enter_context(
             _get_vm(
                 vm_hostname, ignore_reserved=ignore_reserved,
-                ignore_retired=True,
+                allow_retired=True,
```

Comment:
`ignore_reserved` is talking about the hv and `allow_retired` about the vm. Maybe we can call them `ignore_reserved_

I propose renaming `ignore_reserved` into `allow_reserved_hv` and `allow_retired` into `allow_retired_vm`.

Reply:
I think there is something very wrong with the code. `ignore_reserved` passed to `_get_vm()` has no effect on anything at all. I will just rip it out.

Reply:
Ripped out.